app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -223,6 +223,16 @@ export default function () {
         });
       }
     );
+
+    this.route("admin.reportsPage", {
```

Comment:
We don't need `Page` in the route names, these can just be `admin.reports` and `admin.whatsNew` :)